health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadData.js

```
@@ -1015,142 +1057,200 @@
+          <PopUp
+            type={"default"}
+            className={"popUpClass"}
+            footerclassName={"popUpFooter"}
+            heading={
+              type === "boundary"
+                ? t("ES_CAMPAIGN_UPLOAD_BOUNDARY_DATA_MODAL_HEADER")
+                : type === "facilityWithBoundary"
+                ? t("ES_CAMPAIGN_UPLOAD_FACILITY_DATA_MODAL_HEADER")
+                : t("ES_CAMPAIGN_UPLOAD_USER_DATA_MODAL_HEADER")
+            }
+            children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)